.travis.yml

```diff
@@ -1,19 +1,20 @@
 ---
 dist: xenial
 language: python
-python: "3.8"
+python: "3.10"
 os: linux
 services:
   - docker
 install:
-  - pip install pipenv
+  - pip install -I pipenv
```

Comment:
I think I would advise against this flag unless it is needed: `Ignore the installed packages, overwriting them. This can break your system if the existing package is of a different version or was installed with a different package manager!`